src/interceptors/ClientRequest/NodeClientRequest.ts

```diff
@@ -48,6 +48,7 @@ export class NodeClientRequest extends ClientRequest {
   }> = []
   private responseSource: 'mock' | 'bypass' = 'mock'
   private capturedError?: NodeJS.ErrnoException
+  private afterConnectionPhase: boolean = false
```

Comment:
Why not change the default to `pass`?

Reply:
Because that'd mean for the class that it should treat this request as bypassed. Frankly, it should be neither until the request listener promise has been settled. So introducing something like `unknown` is a great internal improvement.

Reply:
Fixed